compiler/rustc_mir/src/interpret/validity.rs

```diff
@@ -104,7 +105,7 @@ pub enum PathElem {
     Field(Symbol),
     Variant(Symbol),
     GeneratorState(VariantIdx),
-    CapturedVar(Symbol),
+    CapturedPlace(Symbol),
```

Comment:
I'd personally not change the name until we have figured out what this should look like